scripts/secnetperf-helpers.psm1

```diff
@@ -222,6 +222,14 @@ function Cleanup-State {
             if ($null -ne (Get-Service msquicpriv -ErrorAction Ignore)) { throw "msquicpriv still running remotely!" }
         }
     }
+
+    # Clean up any ETL residue.
+    try { .\scripts\log.ps1 -Cancel }
```

Comment:
I'm wondering if your first commit is still a good idea. Maybe the `-Cancel` should also clean up old files too in that directory (in log.ps1)?

Reply:
Right, I agree we should contain the cleanup logic in 1 place. I can open a separate PR for log.ps1.

Reply:
https://github.com/microsoft/msquic/pull/4223